src/components/Footer.tsx

```diff
@@ -140,6 +57,8 @@ const Footer: React.FC<IProps> = () => {
 
   const isPageRightToLeft = isLangRightToLeft(intl.locale as Lang)
 
+  const [smallBp] = useToken("breakpoints", ["sm"])
```

Comment:
I agree with this but I think this should be `md` instead of `sm`
```suggestion
  const [mediumBp] = useToken("breakpoints", ["md"])
```